bibtex-actions.el

```diff
@@ -234,12 +234,31 @@ If FORCE-REBUILD-CACHE is t, force reloading the cache."
     (bibtex-actions-refresh force-rebuild-cache))
   bibtex-actions--candidates-cache)
 
+(defun bibtex-actions-complete-key-at-point-oc ()
+    "Complete org-cite citation key at point.
+
+When inserting '@' in a buffer the capf UI will present user with
+a list of entries, from which they can narrow against a string
+which includes title, author, etc., and then select one.  This
+function will then return the key 'key', resulting in '@key' at
+point."
+    ;; FIX exit-function is wrong; results in "no match"
+    ;; TODO don't need a regex for org-cite; what to use instead?
+  (when (and (eq major-mode 'org-mode) (looking-back "@[a-zA-Z]+" 5))
```

Comment:
@ilupin - since you figured out the org-cite code for your at-point, do you know how I would adapt that to here?

I added the org-mode check of course.